pkg/envvar/env_var.go

```diff
@@ -0,0 +1,73 @@
+// envvar package helps converting name/value pairs to devfile EnvVars
+package envvar
+
+import (
+	"errors"
+	"strings"
+
+	devfilev1 "github.com/devfile/api/v2/pkg/apis/workspaces/v1alpha2"
+)
+
+// EnvVar represents an environment variable
+type EnvVar struct {
+	Name  string `yaml:"Name"`
+	Value string `yaml:"Value"`
+}
```

Comment:
Why did you define a separate struct if we already have devfile EnvVar? Is it because of the yaml keys?

Reply:
Yes, it seems it has been done to have `Name` and `Value` written capitalized in the output of `odo config view`.